x-pack/solutions/search/plugins/search_synonyms/public/search_synonyms_router.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Route, Routes } from '@kbn/shared-ux-router';
+import React from 'react';
+import { SynonymsSetDetail } from './components/synonyms_set_detail/synonyms_set_detail';
+import { SearchSynonymsOverview } from './components/overview/overview';
+
+export const SearchSynonymsRouter = () => {
+  return (
+    <Routes>
+      <Route exact path="/:synonymsSetId">
```

Comment:
should we have an additional slug in this path, like `/sets/:synonymsSetId` in case we need other routes here in the future? I'm not sure how likely that is though.

Reply:
Routes are designed to be as close to the Elasticsearch APIs. Not necessarily required though. 
i.e. `GET /_synonyms/{set_id}/{rule_id}`